h2/src/main/org/h2/mvstore/db/MVTable.java

```diff
@@ -49,6 +49,14 @@
  */
 public class MVTable extends TableBase {
 
+    private static final String TRACE_LOCK_OK = "ok";
+    private static final String TRACE_LOCK_WAITING_FOR = "waiting for";
+    private static final String TRACE_LOCK_REQUESTING_FOR = "requesting for";
+    private static final String TRACE_LOCK_TIMEOUT_AFTER = "timeout after ";
+    private static final String TRACE_LOCK_UNLOCK = "unlock";
+    private static final String TRACE_LOCK_ADDED_FOR = "added for";
+    private static final String TRACE_LOCK_ADD_UPGRADED_FOR = "add (upgraded) for ";
```

Comment:
the change here in this file doesn't seem useful

Reply:
These literals have been gathered as constants, instead of being used individually in situ.
This allows us to use the strings as "event type enums". I found it useful for adding more debugging code when I was interested om specific locks and lock events.

I could make them into actual lock event type enums... ?